### DISCUSSION TITLE
repocket stopped working

### DISCUSSION THREAD

**Discussion by peercoder** [engageub/InternetIncome#120] [General]
Hi,
repocket is working but when i delete and start containers repocket stopped working
repocket containers still running but not earning
container logs:
{ data: { message: 'Peer not found.' }, status: 406 }
How fix this error?
Thanks!

**Comment by engageub** [comment]
Hello,
Repocket [docker image](https://hub.docker.com/r/repocket/repocket/tags)  was updated 19 hours ago. 
Could you please let me know if you are using with proxies or direct connection.
I just logged in through play with docker using direct connection and is working fine.
Official docker image is being used in the script.

Latest version works here https://labs.play-with-docker.com/


Thank you

**Reply by peercoder** [reply]
Hi,
not working with proxies and direct connection
repocket container logs:

peer-client-version: 1.1.41
Your current version is : 1.1.34
Wired
process.platform : linux
docker-fd7779916b648b48
PeerService(9064) -> creating peer userId -> iIFmyZfjm9V7fUoS3dbVYV0M5LP2
PeerService(9064) -> creating peer response -> 200
start connection
P2PS-MainSocket -> Authentication
PeerService(9064) -> Connection established
markPeerAsAlive -> start -  663f0a1a91b15100122cbdad
markPeerAsAlive -> isAlive
peer connected
PeerMonitor -> onInit
ConnectionMonitor -> onInit
req: 
new socket req - _PKPZAqRffcd
GET https://verification.repocket.com/api/peer/verification HTTP/1.1
Host: verification.repocket.co


Accept: */*
Connection: close
User-Agent: curl/7.79.1
 RequestHandlerSocket ->  onClose =>  _PKPZAqRffcd
Connection monitor is running
Internet connection is present
peer monitor is running iIFmyZfjm9V7fUoS3dbVYV0M5LP2 / 663f0a1a91b15100122cbdad
Connection monitor is running
Internet connection is present
PeerMonitor -> Monitor Stopped.
PeerService(9064) -> _handleConnectionClosed
peer disconnected
PeerService(9064) -> resetPeer
PeerService(9064) -> Stop
ConnectionMonitor -> Monitor Stopped.
VPNWatcher -> stop
peer disconnected
{ data: { message: 'Peer not found.' }, status: 406 }
P2PService -> CLOSE SOCKET CONNECTION
main socket destroy
peer resetting
MainSocket -> close false
doneHandler false
main socket dont renew connection cause of error

Thanks for your support!

**Reply by engageub** [reply]
The above logs show peer connected. Please wait for few minutes and check your dashboard.

**Reply by engageub** [reply]
If you are using direct connection. Could you please check with Repocket support Team.
Please find their discord channel. Just mention these logs in the help section. 

https://discord.gg/6Y3y6mJD

Make sure there are not more than 2 instances running on the same IP address. 

Official docker image is being used. If there is any issue, it has to resolved by repocket team. 

Thank you

**Reply by engageub** [reply]
If you want to use the older version instead of latest version which was updated today, you will have to change the following in the internetIncome.sh file.

Open internetIncome.sh file and replace repocket/repocket with repocket/repocket:1.1.33 

There are 2 places where this has to be changed. One at line number 448 and the other at line number 450.

https://github.com/engageub/InternetIncome/blob/main/internetIncome.sh

Thank you

**Reply by engageub** [reply]
I had a look at the discord group where one of the user was getting the same error on 12/08/2023.

https://discord.com/channels/953847091736760320/1034409095995871272/1181868374876237834

They suggested to relaunch the app again.
Later, they suggested to install the OS with 64 bit on raspberry PI instead of 32 bit.

But it was working for you till yesterday. So there might be problem with latest image or some caching issue after the new image is pulled. 


Thank you

**Reply by peercoder** [reply]
Thank you so much, I switched to older version 1.1.33, it is working

Can not add new device on the latest version, logs show peer connected but then peer disconnected
{ data: { message: 'Peer not found.' }, status: 406 }
So it not updated earnings and number devices on repocket dashboard

I noticed the different in container logs is verification on the latest version may be cause peer disconnected:
GET https://verification.repocket.com/api/peer/verification HTTP/1.1

Thanks!

**Reply by engageub** [reply]
Hello,
Repocket has been updated again 2 days back in dockerhub. You may try the latest version and see if the issue still persists.

Thank you
